doc/dev/quick_start.rst

```diff
@@ -742,11 +745,30 @@ as well as vectorized using ``jax.vmap``:
 >>> jax.vmap(cost_fn)(jnp.array([[0.1, 0.2, 0.3], [0.4, 0.5, 0.6]]))
 Array([1.32269195, 1.53905377], dtype=float64)
 
+
 In particular, this allows for a reduction in boilerplate when using
-JAX-compatible optimizers such as ``jaxopt``:
+JAX-compatible optimizers such as ``optax``.
```

Comment:
@tzunghanjuang I now see why you deleted this; it's actually no less boilerplate 😆 

Feel free to delete this section if you think it makes sense.

Reply:
Thank you. I have removed this part.